deployments-ropsten.json

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
I'm confirming I diffed this file against the ropsten section of the original deployments.json and found no changes.